### PR TITLE
added "next_actions" to learn() and SimpleSARSA algorithm

### PR DESCRIPTION
**Pull request by yu239**
This is a preparation for implementing Successor Representation. On the "CartPole-v0" problem, SimpleSARSA seems to converge even faster than SimpleAC (because of fewer parameters?)

### CODE REVIEW

File: flare/framework/tests/test_simple_games.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from flare.framework.computation_task import ComputationTask
-from flare.algorithm_zoo.simple_algorithms import SimpleAC, SimpleQ
+from flare.algorithm_zoo.simple_algorithms import SimpleAC, SimpleQ, SimpleSARSA
+from flare.algorithm_zoo.successor_representation import SuccessorRepresentationQ
 from flare.model_zoo.simple_models import SimpleModelAC, SimpleModelQ, GaussianPolicyModel
+from flare.model_zoo.successor_representation_models import SimpleSRModel
```
Comment (yu239) [inline flare/framework/tests/test_simple_games.py]: remove this

File: flare/framework/tests/test_simple_games.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from flare.framework.computation_task import ComputationTask
-from flare.algorithm_zoo.simple_algorithms import SimpleAC, SimpleQ
+from flare.algorithm_zoo.simple_algorithms import SimpleAC, SimpleQ, SimpleSARSA
+from flare.algorithm_zoo.successor_representation import SuccessorRepresentationQ
```
Comment (yu239) [inline flare/framework/tests/test_simple_games.py]: remove this

File: flare/framework/tests/test_simple_games.py [hunk 3]
```diff
@@ -63,26 +87,28 @@ def test_gym_games(self):
                     model=SimpleModelQ(
                         dims=state_shape,
                         num_actions=num_actions,
-                        mlp=nn.Sequential(mlp, nn.Linear(128, num_actions))),
-                    exploration_end_steps=1000000,
+                        mlp=nn.Sequential(
+                            mlp, nn.Linear(hidden_size, num_actions))),
+                    exploration_end_steps=500000,
```
Comment (yu239) [inline flare/framework/tests/test_simple_games.py]: merge the two SimpleModelQ code blocks.

File: flare/framework/tests/test_simple_games.py [hunk 4]
```diff
@@ -36,22 +47,32 @@ def test_gym_games(self):
         for game, threshold, on_policy, discrete_action in \
```
Comment (skylian) [inline flare/framework/tests/test_simple_games.py]: use a list of algorithms, instead of on_policies' anddiscrete_actions`

**Pull request merged**